test/build.info

```diff
@@ -363,20 +363,6 @@ IF[{- !$disabled{tests} -}]
   INCLUDE[recordlentest]=../include ../apps/include
   DEPEND[recordlentest]=../libcrypto ../libssl libtestutil.a
 
-  SOURCE[drbgtest]=drbgtest.c
-  INCLUDE[drbgtest]=../include ../apps/include
-  DEPEND[drbgtest]=../libcrypto.a libtestutil.a
-
-  SOURCE[drbg_cavs_test]=drbg_cavs_test.c drbg_cavs_data_ctr.c \
-                         drbg_cavs_data_hash.c drbg_cavs_data_hmac.c
-
-  INCLUDE[drbg_cavs_test]=../include ../apps/include . ..
-  DEPEND[drbg_cavs_test]=../libcrypto libtestutil.a
-
-  SOURCE[drbg_extra_test]=drbg_extra_test.c
-  INCLUDE[drbg_extra_test]=../include . .. ../apps/include
-  DEPEND[drbg_extra_test]=../libcrypto libtestutil.a
-
```

Comment:
Some of the tests (e.g. `drbg_cavs_test`, `drbg_extra_test`) might be redundant, but what about all the tests in `drbgtest`? Do they have appropriate replacements? If not, can they really all be dropped?